### DISCUSSION TITLE
Creating a Python site using EE

### DISCUSSION THREAD

**Discussion by JacobCeccRoos** [EasyEngine/easyengine#1775] [Support Requests]
I am looking at building a web frontend to manage my EE server (this would be for personal usage). Anyone know what it would take to run a python app site from EE, would I just have to do the standard `ee site create sitename --type= ` could I specify python in the type or would I need to build a custom type first?

**Comment by erickrex** [comment]
It is feasible... in theory. But it does not make sense, unless you have deep knowledge of docker, uvicorn (the python webserver preferred over nginx), bash, databases and in general a very holistic understanding of the backend. Someone that has all of these skills, could add support for some basic python applications to easyengine after some months of hard work, probably.
